src/main/java/com/example/management/controller/LoginDispatcher.java

```diff
@@ -0,0 +1,20 @@
+package com.example.management.controller;
+
+import java.io.IOException;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+public class LoginDispatcher extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        doPost(request, response);
```

Comment:
basically doGet and doPost are working differently, one bad scenario that  doPost change and may effect to doGet. In that case, people have to change code -> violate open & close principle ( part of solid principles)  
 
 So it's better to have separate method, with the purpose of forward, redirecting page. 
 
 Another, can I use redirect in this case instead of forward page?

Reply:
Because I used an `<a>` tag pretending to be a button to access this servlet for the demo, I had to provide a doGet method. When the MoviePage is completed, this method will not  be needed.